build.cmd

```diff
@@ -1,40 +1,40 @@
-@echo off
-cd %~dp0
-
+@ECHO off
 SETLOCAL
+
+SET REPO_FOLDER=%~dp0
+CD %REPO_FOLDER%
+
+SET BUILD_FOLDER=.build
+SET KOREBUILD_FOLDER=%BUILD_FOLDER%\KoreBuild-dotnet
+SET KOREBUILD_VERSION=
+
+SET NUGET_PATH=%BUILD_FOLDER%\NuGet.exe
```

Comment:
I'm going to answer a few of your questions here. Most of the changes here are because of the new KoreBuild-dotnet. The new build structure is:

- Everything build related goes in `.build` instead of `.nuget` and `packages`. So now we have `.build\nuget.exe` and `.build\KoreBuild-dotnet`
- Sake is now part of KoreBuild-dotnet (as a dependecy)
- The `build.cmd` and `build.sh` files do almost nothing. They just download `KoreBuild-dotnet` and invoke a script in it